src/shared/config/ColumnConfiguration.tsx

```diff
@@ -0,0 +1,68 @@
+import TaxonomyView from '../components/entry/TaxonomyView';
+import AccessionView from '../components/results/AccessionView';
+
+import getLabelAndTooltip from '../utils/getLabelAndTooltip';
+
+import AnnotationCovered from '../../automatic-annotations/shared/column-renderers/AnnotationCovered';
+import TaxonomicScope from '../../automatic-annotations/shared/column-renderers/TaxonomicScope';
+import { TaxonomyDatum } from '../../supporting-data/taxonomy/adapters/taxonomyConverter';
+import { ARBAAPIModel } from '../../automatic-annotations/arba/adapters/arbaConverter';
+import { Namespace } from '../types/namespaces';
+
+const SharedColumnConfiguration = {
+  organism_id: <Schema,>(
+    getter: (data: Schema) => TaxonomyDatum | undefined
+  ) => ({
+    ...getLabelAndTooltip(
+      'Organism ID',
+      'NCBI taxonomy identifier of the source organism (TaxId)'
+    ),
+    render: (data: Schema) => {
+      const taxonomyData = getter(data);
+      return taxonomyData && <TaxonomyView data={taxonomyData} displayOnlyID />;
+    },
+  }),
+  organism: <Schema,>(getter: (data: Schema) => TaxonomyDatum | undefined) => ({
+    ...getLabelAndTooltip(
+      'Organism',
+      'Scientific name (and synonyms) of the source organism',
+      'organism-name'
+    ),
+    render: (data: Schema) => {
+      const taxonomyData = getter(data);
+      return taxonomyData && <TaxonomyView data={taxonomyData} />;
+    },
+  }),
```

Comment:
That `<Schema,>` got me thinking if it was a typo. I understand this is to avoid this being parsed as JSX. Wondering if the below suggestion would be acceptable in order to avoid this, even though it wouldn't use an arrow function then (equivalent, but not consistent)
```suggestion
  organism<Schema>(getter: (data: Schema) => TaxonomyDatum | undefined) {
    return {
      ...getLabelAndTooltip(
        'Organism',
        'Scientific name (and synonyms) of the source organism',
        'organism-name'
      ),
      render: (data: Schema) => {
        const taxonomyData = getter(data);
        return taxonomyData && <TaxonomyView data={taxonomyData} />;
      },
    };
  },
```